src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -85,13 +89,13 @@ class UserService(val directoryDAO: DirectoryDAO, val cloudExtensions: CloudExte
       groups <- directoryDAO.listUserDirectMemberships(uid, samRequestContext)
       _ <- user.googleSubjectId.traverse { googleSubjectId =>
         for {
-          _ <- directoryDAO.setGoogleSubjectId(uid, googleSubjectId, samRequestContext)
+          _ <- directoryDAO.setGoogleSubjectId(uid, googleSubjectId, samRequestContext).withDebugLogMessage(s"Adding user $uid to directory with Google Subject ID: $googleSubjectId")
```

Comment:
```suggestion
          _ <- directoryDAO.setGoogleSubjectId(uid, googleSubjectId, samRequestContext).withDebugLogMessage(s"Updating User: $uid - setting Google Subject ID: $googleSubjectId")
```

Feel free to wordsmith this differently.  The point we want to convey is that we are setting the Google Subject ID for an already existing user.